apps/admin/src/app/dashboard/reports/[id]/page.tsx

```diff
@@ -12,7 +12,7 @@ import { SolutionReport } from './_components/solution.report';
 import { UserReport } from './_components/user.report';
 import { ReportActions } from './actions';
 
-export interface Props {
+export interface ReportPageProps {
```

Comment:
[Explanation] The Vercel style guide has [@typescript-eslint/naming-convention](https://typescript-eslint.io/rules/naming-convention) configured to block `Props` and `State` as interface names. I personally agree with and like this convention: component-specific names are more clear to work with. Especially if you end up exporting and using them cross-file.